pkg/cloud/services/ec2/routetables.go

```diff
@@ -92,7 +98,13 @@ func (s *Service) reconcileRouteTables() error {
 			return err
 		}
 
-		if err := s.associateRouteTable(rt, sn.ID); err != nil {
+		if err := wait.WaitForWithRetryable(wait.NewBackoff(), func() (bool, error) {
+			if err := s.associateRouteTable(rt, sn.ID); err != nil {
+				return false, err
+			}
+			return true, nil
+		}, awserrors.RouteTableNotFound, awserrors.SubnetNotFound); err != nil {
+			record.Warnf(s.scope.Cluster, "FailedAssociateRouteTable", "Failed to associate managed RouteTable %q with Subnet %q: %v", rt.ID, sn.ID, err)
```

Comment:
I think this failure might be reported in assocationRouteTable() now

Reply:
Correct - safe to remove from here

Reply:
yep